### PR TITLE
#3184 Refactor and add the ability to toggle the toolbox

### PR DESCRIPTION
**Pull request by konekoya**
### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #3184)
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/client/src/components/Layout/Styles.js [hunk 1]
```diff
@@ -14,66 +14,70 @@
...
-    margin-top: ${props => props.theme.spacing(2)}px;
-    margin-bottom: ${props => props.theme.spacing(3)}px;
-  }
-`;
-
-export const WorkspaceList = styled.ul`
-  display: flex;
-  align-items: center;
-  flex-direction: column;
+export const Header = styled.header(
```
Comment (konekoya) [inline ohara-manager/client/src/components/Layout/Styles.js]: Refactoring

File: ohara-manager/client/src/components/Pipeline/Toolbox/Toolbox.js [hunk 2]
```diff
@@ -141,43 +81,37 @@ const Toolbox = () => {
             </div>
           </ExpansionPanelDetails>
         </ExpansionPanel>
-        <ExpansionPanel
-          square
-          expanded={expanded === 'sink-panel'}
-          onChange={handleChange('sink-panel')}
-        >
+        <ExpansionPanel square expanded={expanded.streamApp}>
```
Comment (konekoya) [inline ohara-manager/client/src/components/Pipeline/Toolbox/Toolbox.js]: Refactor so it can be used in the toolbar as well...

**Comment by konekoya** [comment]
@jackyoh I think the fail alert is not properly pinging fontend devs...

**Comment by konekoya** [comment]
@oharastream/developer

**Comment by chia7712** [comment]
@oharastream/frontend

**Comment by chia7712** [comment]
@oharastream/frontend

**Comment by chia7712** [comment]
This PR may break API compatibility!!! ping @oharastream/frontend see https://builds.is-land.com.tw/job/PreCommit-OHARA/6794/ for logs

**Pull request merged**